pkg/util/validation/limits.go

```diff
@@ -230,7 +229,7 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	f.BoolVar(&l.CompactorBlockUploadEnabled, "compactor.block-upload-enabled", false, "Enable block upload API for the tenant.")
 
 	// Query-frontend.
-	f.Var(&l.MaxTotalQueryLength, maxTotalQueryLengthFlag, fmt.Sprintf("Limit the total query time range (end - start time). This limit is enforced in the query-frontend on the received query. Defaults to the value of -%s if set to 0.", maxQueryLengthFlag))
+	f.Var(&l.MaxTotalQueryLength, maxTotalQueryLengthFlag, fmt.Sprintf("Limit the total query time range (end time minus start time). This limit is enforced in the query-frontend on the received query. Defaults to the value of `-%s` if set to 0.", maxQueryLengthFlag))
```

Comment:
Please remove the backtick. It will not get rendered as we expect, cause the whole config doc is already rendered inside a big code block.

```suggestion
	f.Var(&l.MaxTotalQueryLength, maxTotalQueryLengthFlag, fmt.Sprintf("Limit the total query time range (end time minus start time). This limit is enforced in the query-frontend on the received query. Defaults to the value of -%s if set to 0.", maxQueryLengthFlag))
```

Reply:
👍  thanks for the hint, and FYI @osg-grafana